app/javascript/dashboard/components/widgets/conversation/ConversationAdvancedFilter.vue

```diff
@@ -48,9 +48,9 @@
 <script>
 import alertMixin from 'shared/mixins/alertMixin';
 import { required, requiredIf } from 'vuelidate/lib/validators';
-import FilterInputBox from './components/FilterInput.vue';
+import FilterInputBox from '../FilterInput.vue';
 import languages from './advancedFilterItems/languages';
-import countries from './advancedFilterItems/countries';
+import countries from '../../../../shared/constants/countries';
```

Comment:
We could use absolute path here like `app/javascript/dashboard/components/MyLib`

Reply:
Fixed